### PR TITLE
V0.6.1

### PR DESCRIPTION
**Pull request by oguzhanunlu**

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+gopkg.in/fsnotify.v1 v1.4.7/go.mod h1:Tz8NjZHkW78fSQdbUxIjBTcgA1z1m8ZHf0WmKUhAMys=
+gopkg.in/mcuadros/go-syslog.v2 v2.2.1/go.mod h1:l5LPIyOOyIdQquNg+oU6Z3524YwrcqEm0aKH+5zpt2U=
+gopkg.in/natefinch/lumberjack.v2 v2.0.0 h1:1Lc07Kr7qY4U2YPouBjpCLxpiyxIVoxqXgkXLknAOE8=
+gopkg.in/natefinch/lumberjack.v2 v2.0.0/go.mod h1:l0ndWWf7gzL7RNwBG7wST/UCcT4T24xpD6X8LsfU/+k=
+gopkg.in/square/go-jose.v2 v2.2.2 h1:orlkJ3myw8CN1nVQHBFfloD+L3egixIa4FvUP6RosSA=
+gopkg.in/square/go-jose.v2 v2.2.2/go.mod h1:M9dMgbHiYLoDGQrXy7OpJDJWiKiU//h+vD76mk0e1AI=
+gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7/go.mod h1:dt/ZhP58zS4L8KSrWDmTeBkI65Dw0HsyUHuEVlX15mw=
+gopkg.in/yaml.v2 v2.2.1/go.mod h1:hI93XBmqTisBFMUTm0b8Fm+jr3Dg1NNxqwp+5A1VGuI=
+gopkg.in/yaml.v2 v2.2.2 h1:ZCJp+EgiOT7lHqUV2J862kp8Qj64Jo6az82+3Td9dZw=
+gopkg.in/yaml.v2 v2.2.2/go.mod h1:hI93XBmqTisBFMUTm0b8Fm+jr3Dg1NNxqwp+5A1VGuI=
```
Comment (BenFradet) [inline go.sum]: what's the purpose of both these files? should they be committed or not

File: provisioning/roles/sp_mini_3_build_go_projects/tasks/main.yml [hunk 2]
```diff
@@ -40,42 +40,44 @@
...
-- name: Build Caddy
-  environment:
-    PATH: "{{ lookup('env','PATH') }}:{{ go_bin }}"
-    GOPATH: "{{ go_path }}"
-  shell: "cd $GOPATH/src/github.com/mholt/caddy/caddy; go run build.go -goos=linux -goarch=amd64"
+# - name: Build Caddy
+#   environment:
+#     PATH: "{{ lookup('env','PATH') }}:{{ go_bin }}"
+#     GOPATH: "{{ go_path }}"
+#   shell: "cd $GOPATH/src/github.com/mholt/caddy/caddy; go run build.go -goos=linux -goarch=amd64"
```
Comment (BenFradet) [inline provisioning/roles/sp_mini_3_build_go_projects/tasks/main.yml]: shoudl all those commented lines be removed?

File: go.sum (hunk 1 above)
Comment (oguzhanunlu) [reply go.sum]: Official Go documentation recommends doing so.

From https://github.com/golang/go/wiki/Modules#releasing-modules-all-versions 
> Ensure your go.sum file is committed along with your go.mod file.

File: provisioning/roles/sp_mini_3_build_go_projects/tasks/main.yml (hunk 2 above)
Comment (oguzhanunlu) [reply provisioning/roles/sp_mini_3_build_go_projects/tasks/main.yml]: Yeah, I'll remove them.

File: go.sum (hunk 1 above)
Comment (oguzhanunlu) [reply go.sum]: wdyt @BenFradet ? should I proceed with the release?

Comment (BenFradet) [reply go.sum]: if it's a good practice I'm okay with it :+1:

**Comment by oguzhanunlu** [comment]
hey @BenFradet , I needed to include additional work and update some existing commits after your review. Do you mind taking another look at the PR? I also published `0.6.1-rc1` to both AWS & GCP.

**Comment by oguzhanunlu** [comment]
Thanks @BenFradet , yes, we've changed the way we publish artifacts, the process is moved to Travis and `BIN.enc` is an encrypted file which is decrypted during build process and used by Packer when publishing to GCP.

**Pull request closed without merging**